syntax = "proto3";

package systemmonitor;

message CpuUsageRequest {
}

message CpuUsageResponse {
  float usage_percent = 1;
}

service SystemMonitor {
    rpc GetCpuUsage (CpuUsageRequest) returns (CpuUsageResponse) {}
}